package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "js-reporters",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "esperanto -b -i lib/index.js -t umd --name JsReporters | babel -o dist/js-reporters.js --blacklist strict -m umd",
+    "lint": "eslint lib/"
+  },
+  "devDependencies": {
+    "babel": "^5.6.14",
+    "eslint": "^0.24.1",
+    "esperanto": "^0.7.3",
+    "jasmine-core": "^2.3.4",
+    "qunitjs": "^1.18.0"
+  },
+  "eslintConfig": {
```

Comment:
This doesn't seem to include any style checks by default. Let's either add those or add jscs with a preset. eslint doesn't seem to bundle any kind of preset and seems to have no plans to add any either.